getting-started/multiband-analysis.rst

```diff
@@ -63,39 +70,42 @@ From the source table's schema you know this flux has units of counts:
    iSources.getSchema().find('base_PsfFlux_instFlux').field.getUnits()
 
 Transforming this instrumental flux into a magnitude requires knowing the coadd's photometric calibration, which you can get from the coadd dataset.
-The coadd you made in :doc:`part 4 <coaddition>` with :command:`assembleCoadd.py` doesn't have calibration info attached to it, though.
-Instead, you want the ``deepCoadd_calexp`` dataset, which was created by the :command:`detectCoaddSources.py` command-line task, because it does have calibrations.
-You can access these calibrations directly from ``deepCoadd_calexp_photoCalib`` datasets for each filter:
+The coadd you made in :doc:`part 5 <coaddition>` with the ``assembleCoadd`` pipeline doesn't have calibration info attached to it, though.
+Instead, you want the ``deepCoadd_calexp`` dataset, which was created by the ``coadd_measurement`` pipeline, because it does have calibrations.
+You can access these calibrations directly from ``deepCoadd_calexp.photoCalib`` datasets for each filter:
 
 .. code-block:: python
 
-   rCoaddPhotoCalib = butler.get('deepCoadd_calexp_photoCalib',  {'filter': 'HSC-R', 'tract': 0, 'patch': '1,1'})
-   iCoaddPhotoCalib = butler.get('deepCoadd_calexp_photoCalib',  {'filter': 'HSC-I', 'tract': 0, 'patch': '1,1'})
+   gCoaddPhotoCalib = butler.get('deepCoadd_calexp.photoCalib', band='g', tract=9813, patch=41)
+   rCoaddPhotoCalib = butler.get('deepCoadd_calexp.photoCalib', band='r', tract=9813, patch=41)
+   iCoaddPhotoCalib = butler.get('deepCoadd_calexp.photoCalib', band='i', tract=9813, patch=41)
 
 .. note::
 
    An alternative way to get the ``lsst.afw.image.PhotoCalib`` object is from the ``deepCoadd_calexp`` dataset object:
 
    .. code-block:: python
 
-      rCoaddCalexp = butler.get('deepCoadd_calexp',  {'filter': 'HSC-R', 'tract': 0, 'patch': '1,1'})
+      rCoaddCalexp = butler.get('deepCoadd_calexp', band='r', trct=9813, patch=41)
       rCoaddPhotoCalib = rCoaddCalexp.getPhotoCalib()
 
 These ``PhotoCalib`` objects not only have methods for directly accessing calibration information, but also for applying those calibrations.
```

Comment:
Maybe say that reading the entire calexp just to obtain the photo calib is slower than doing the explicit component read in butler.